packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js

```diff
@@ -16,6 +16,9 @@ export default class PciInstanceController {
 
   $onInit() {
     this.loadMessages();
+    this.PciProjectsProjectInstanceService.getSupportLevel().then((value) => {
+      this.supportLevel = `pci_projects_project_instances_instance_support_level_${value.level}`;
```

Comment:
**question:** Isn't this supposed to be translation?
```suggestion
      this.supportLevel = this.$translate.instant(`pci_projects_project_instances_instance_support_level_${value.level}`);
```

Reply:
It is. Thank your for noticing.